src/logic/operate.js

```diff
@@ -6,9 +6,9 @@ const operate = (num1, num2, operation) => {
   switch (operation) {
     case '+':
       return numb1.plus(numb2).toString();
-    case '&times;':
+    case 'x':
       return numb1.times(numb2).toString();
-    case '&minus;':
+    case '-':
       return numb1.minus(numb2).toString();
     case '%':
```

Comment:
**we need to work on the operator `% `**
the operator seems not working.

When it comes to `%` we only need one operand.
example: click `10` then `%` = 0.01

example: `num1 = (num1 *= 0.01).toString();`